### PR TITLE
Add rubocop [WIP]

### PR DESCRIPTION
**Pull request by rohitpaulk**
`29 files inspected, 613 offenses detected`

### CODE REVIEW

**Comment by rohitpaulk** [comment]
Down to `29 files inspected, 49 offenses detected`, and our builds are just 36 seconds long :dancer:

**Comment by sarupbanskota** [comment]
woot - that's a lot less offenses from the earlier one! :)

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -47,19 +48,22 @@ def destroy
...
+      redirect_to root_url if current_user.nil?
     end
 
     def comment_params
-      params.require(:comment).permit(:polycomment_id,:polycomment_type,:issue,:body)
+      params.require(:comment).permit(
+        :polycomment_id,
+        :polycomment_type,
+        :issue,
+        :body
```
Comment (sarupbanskota) [inline app/controllers/comments_controller.rb]: awesome, I find this way of doing it so much more easier to work with.

I remember once just cleaning up as much occurrence of parameter formatting as I could find, without which, my eyes would just.. refuse to work :D

**Comment by sarupbanskota** [comment]
thanks for this man! I'll be following it, but let me know when it's ready to merge anyway.

**Comment by rohitpaulk** [comment]
@sarupbanskota - Ready to merge.

File: app/mailers/notif_mailer.rb [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
     @notification = notification
 
-    @subjecttext = notification.actor.username + notification.messageverb + notification.objectname
-   	mail(to: @victim.email, subject: @subjecttext)
-
-    #notification_path(notification)
+    @subjecttext = notification.actor.username + notification.messageverb
+                  + notification.objectname
+    mail(to: @victim.email, subject: @subjecttext)
   end
```
Comment (sonalkr132) [inline app/mailers/notif_mailer.rb]: You should use:
```Ruby 
@subjecttext = notification.actor.username + notification.messageverb + \
                         notification.objectname
```
Current version gives error:
```
NoMethodError - undefined method `+@' for "wewef":String:
```
here, "wewef" is objectname.

Comment (rohitpaulk) [inline app/mailers/notif_mailer.rb]: @sonalkr132 - Aah, please send in a PR with the fix?

Comment (sonalkr132) [inline app/mailers/notif_mailer.rb]: #270

**Pull request merged**